go-controller/pkg/ovn/egressip.go

```diff
@@ -1874,6 +1886,12 @@ func (e *egressIPController) addPodEgressIPAssignment(egressIPName string, statu
 // deletePodEgressIPAssignment deletes the OVN programmed egress IP
 // configuration mentioned for addPodEgressIPAssignment.
 func (e *egressIPController) deletePodEgressIPAssignment(egressIPName string, status egressipv1.EgressIPStatusItem, podIPs []*net.IPNet) error {
+	start := time.Now()
+	defer func() {
+		end := time.Since(start)
```

Comment:
Also, I think this should be level 5 log output. I maybe not understanding why you want to record the latency.

Reply:
We will keep this as a metric, but will disable it by default, as it is only expected to be useful for egress scaling tests.

Reply:
Disabled by default. To enable this metric, use newly added flag: `--metrics-enable-eip-scale`